model/Extension/Classes/CdxPropertiesExtension.md

```diff
@@ -11,7 +11,7 @@ A type of extension consisting of a list of name value pairs.
 This extension provides a more structured extension using a name-value
 approach.
 
-Unlike key-value stores, cdxProperties support duplicate names, each
+Unlike key-value stores, cdxProperty support duplicate names, each
```

Comment:
Why no re-use the same wording as elsewhere?

Reply:
Because cdxProperty is presented here in CdxPropertiesExtension, so it can be referred to directly.

Elsewhere, they don't have cdxProperty, so I have to go indirect. Can change to prefer similarity over directness.